File: app/src/main/java/org/mifos/mobile/ui/views/RegistrationVerificationView.java
package org.mifos.mobile.ui.views;

import org.mifos.mobile.ui.views.base.MVPView;

/**
 * Created by dilpreet on 31/7/17.
 */

public interface RegistrationVerificationView extends MVPView {

    void showVerifiedSuccessfully();

    void showError(String msg);
}

File: app/src/main/java/org/mifos/mobile/ui/views/RegistrationVerificationView.kt
package org.mifos.mobile.ui.views

import org.mifos.mobile.ui.views.base.MVPView

/**
 * Created by dilpreet on 31/7/17.
 */
interface RegistrationVerificationView : MVPView {
    fun showVerifiedSuccessfully()
    fun showError(msg: String?)
}